bin/scrape_software_versions.py

```diff
@@ -54,7 +76,7 @@
 section_name: 'nf-core/eager Software Versions'
 section_href: 'https://github.com/nf-core/eager'
 plot_type: 'html'
-description: 'are collected at run time from the software output.'
+description: 'are collected at run time from all software contained in EAGER2. This may include software not used in your run!'
```

Comment:
```suggestion
description: 'are collected at run time from all software contained in nf-core/eager. This may include software not used in your run!'
```